lib/trailblazer/rails/railtie.rb

```diff
@@ -36,6 +36,7 @@ def self.load_for(app)
     initializer "trailblazer.application_controller" do
       reloader_class.to_prepare do
         ApplicationController.send :include, Trailblazer::Rails::Controller
+        ApplicationController.send :include, Trailblazer::Rails::Controller::Cell if defined?(::Cell)
```

Comment:
I see here (correct me if I'm wrong) the same problem as I've reportd earlier. If I use Rails but for example my base controller is Namespaced::Whatever::ApplicationController it won't integrate as it assumes the existence (and location) of ApplicationController. I know that in Rails 99% of people use ApplicationController (or just keep it empty) but in a particular one project that I developer, there's a case like described and then it will fail :(